pkg/storaged/stratis-details.jsx

```diff
@@ -615,18 +651,16 @@ export const StratisPoolDetails = ({ client, pool }) => {
         };
     }
 
-    const offsets = [];
-    let total = 0;
-    filesystems.forEach(fs => {
-        offsets.push(total);
-        total += fs.Used[0] ? Number(fs.Used[1]) : 0;
-    });
-
-    const rows = filesystems.map((fs, i) => render_fsys(fs, offsets[i], total));
+    const rows = filesystems.map((fs, i) => render_fsys(fs, offsets[i]));
 
     const content = (
         <Card>
-            <CardHeader actions={{ actions: <><StorageButton onClick={create_fs}>{_("Create new filesystem")}</StorageButton></> }}>
+            <CardHeader actions={{
+                actions: <StorageButton onClick={create_fs}
+                                                           excuse={!pool.Overprovisioning && pool_free < fsys_min_size ? _("Not enough space for new filesystems") : null}>
```

Comment:
This looks a bit funky, does `eslint` not complain about the alignment?

Reply:
Yeah, I have fixed it. *shrug*